src/components/services/controllers.tsx

```diff
@@ -72,3 +76,115 @@ export async function viewService(
     ),
   };
 }
+
+export async function listServiceLogs(ctx: IContext, params: IParameters): Promise<IResponse> {
+  const cf = new CloudFoundryClient({
+    accessToken: ctx.token.accessToken,
+    apiEndpoint: ctx.app.cloudFoundryAPI,
+    logger: ctx.app.logger,
+  });
+
+  const [userProvidedServices, space, organization] = await Promise.all([
+    cf.userServices(params.spaceGUID),
+    cf.space(params.spaceGUID),
+    cf.organization(params.organizationGUID),
+  ]);
+
+  const isUserProvidedService = userProvidedServices.some(
+    s => s.metadata.guid === params.serviceGUID,
+  );
+
+  if (isUserProvidedService) {
+    throw new UserFriendlyError(UNSUPPORTED_SERVICE_LOGS_REQUEST);
+  }
+
+  const serviceInstance = await cf.serviceInstance(params.serviceGUID);
+  const service = await cf.service(serviceInstance.entity.service_guid);
+
+  if (!['postgres', 'mysql'].includes(service.entity.label)) {
+    throw new UserFriendlyError(UNSUPPORTED_SERVICE_LOGS_REQUEST);
+  }
+
+  const rds = new RDS();
+  const fileList = await rds.describeDBLogFiles({
+    DBInstanceIdentifier: `rdsbroker-${serviceInstance.metadata.guid}`,
+    MaxRecords: 72,
+  }).promise();
+
+  const template = new Template(
+    ctx.viewContext,
+    `${serviceInstance.entity.name} - Service Metrics Logs`,
+  );
+  template.breadcrumbs = fromOrg(ctx, organization, [
+    {
+      href: ctx.linkTo('admin.organizations.spaces.services.list', {
+        organizationGUID: organization.metadata.guid,
+        spaceGUID: space.metadata.guid,
+      }),
+      text: space.entity.name,
+    },
+    {
+      href: ctx.linkTo('admin.organizations.spaces.services.view', {
+        organizationGUID: organization.metadata.guid,
+        serviceGUID: serviceInstance.metadata.guid,
+        spaceGUID: space.metadata.guid,
+      }),
+      text: serviceInstance.entity.name,
+    },
+    { text: 'Logs' },
+  ]);
+
+  return {
+    body: template.render(<ServiceLogsPage
+      files={(fileList.DescribeDBLogFiles || []).reverse() as ReadonlyArray<IServiceLogItem>}
```

Comment:
```suggestion
      files={(fileList.DescribeDBLogFiles || []) as ReadonlyArray<IServiceLogItem>}
```

Personal preference, but the most recent log file first makes more sense to me

Reply:
`.reverse()` gives us the recent to the top :) See Jani's comment above.